src/internal_events/decoder.rs

```diff
@@ -26,25 +28,65 @@ impl<'a> InternalEvent for DecoderDeserializeFailed<'a> {
 }
 
 #[derive(Debug)]
-pub struct EncoderFramingFailed<'a> {
+pub struct EncoderFramingError<'a> {
     pub error: &'a codecs::encoding::BoxedFramingError,
 }
 
-impl<'a> InternalEvent for EncoderFramingFailed<'a> {
+impl<'a> InternalEvent for EncoderFramingError<'a> {
     fn emit(self) {
         warn!(message = "Failed framing bytes.", error = %self.error, internal_log_rate_secs = 10);
```

Comment:
Perhaps the original warnings + counter should be marked deprecated and eventually removed if we think this is the correct direction?

Reply:
Makes sense to me